server/src/main/java/org/opensearch/identity/IdentityService.java

```diff
@@ -25,9 +26,13 @@ public class IdentityService {
 
     private final Settings settings;
     private final IdentityPlugin identityPlugin;
+    private final ApplicationManager applicationManager;
 
-    public IdentityService(final Settings settings, final List<IdentityPlugin> identityPlugins) {
+    private static IdentityService instance = null;
```

Comment:
Why do we need to reference statics everywhere? All dependencies are injectable and should remain as such